driver/web/docs/resources/admin/application/accounts/system-configs.yaml

```diff
@@ -0,0 +1,73 @@
+get:
+  tags:
+  - Admin
+  summary: Get system configs
+  description: |
+    Returns account system configs
+
+    **Auth:** Requires admin access token with `get_account-system-configs` or `all_account-system-configs` permission
+  security:
+    - bearerAuth: []
+  parameters:
+    - name: id
+      in: path
+      description: The account ID
+      required: true
+      style: simple
+      explode: false
+      schema:
+        type: string
+  responses:
+     200:
+      description: Success
+      content:
+        application/json:
+          schema: 
+            type: object
+            nullable: true
+     400:
+      description: Bad request
+     401:
+      description: Unauthorized
+     500:
+      description: Internal error
+put:
+  tags:
+    - Admin
+  summary: Update account system configs
+  description: |
+    Updates account system configs
+
+    **Auth:** Requires admin access token with `update_account-system-configs` or `all_account-system-configs` permission
+  security:
+    - bearerAuth: []
+  parameters:
+    - name: id
+      in: path
+      description: The account ID
+      required: true
+      style: simple
+      explode: false
+      schema:
+        type: string
+  requestBody:
+    description: Account system configs (will be merged with existing configs)
+    content:
+      application/json:
+        schema: 
+          type: object
+    required: true
+  responses:
+    '200':
+      description: Success
+      content:
+        text/plain:
```

Comment:
We actually aren't necessarily allowing everyone who can update to see the full contents since various parties may use these configs for different purposes and should not necessarily know what others have done with them